js/notifications.js

```diff
@@ -100,8 +100,15 @@
       const last = this.last().toJSON();
       switch (userSetting) {
         case SettingNames.COUNT:
-          title = 'Signal';
-          message = newMessageCountLabel;
+          title = 'Loki Messenger';
+
+          if (last.isFriendRequest) {
+            message = `Friend request ${last.friendRequestType}`;
+          } else if (messagesNotificationCount > 0) {
+            message = newMessageCountLabel;
+          } else {
+            message = '-';
```

Comment:
As discussed offline, we should simply handle this "should never happen" case by returning early?

Reply:
I guess so :)